tests/integration/users_test.go

```diff
@@ -0,0 +1,43 @@
+package tests
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/Real-Dev-Squad/wisee-backend/routes"
+	"github.com/Real-Dev-Squad/wisee-backend/utils"
+	"github.com/uptrace/bun"
+)
+
+var db *bun.DB
+
+func TestMain(m *testing.M) {
```

Comment:
The db.Close() call is only made after the m.Run() call. This means that if any of the tests fail, the database connection will not be closed. This can lead to resource leaks and other problems.

func TestMain(m *testing.M) {
	utils.LoadEnv("../../.env")
	dsn := os.Getenv("TEST_DB_URL")
	db = utils.SetupDBConnection(dsn)

	defer db.Close()

	code := m.Run()

	os.Exit(code)
}